src/backend/routes/user.router.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const router = express.Router();
+
+const { UserController } = require('../api/users/user.controller');
+const { CreateUserInterceptor } = require('../modules/users/interceptor');
```

Comment:
You will have to put all interceptors here, so use restructuring to get interceptors from an array.